prow/plugins/approve/approvers/README.md

```diff
@@ -185,27 +194,33 @@ K8s-bot updates comment:
 	You can indicate your approval by writing `/approve` in a comment
 	You can cancel your approval by writing `/approve cancel` in a comment
 
-K8s-bot merges the PR. It needs to wait its turn in submit queue and passes tests.
+K8s-bot merges the PR, because it has both the **lgtm** and **approved**. It K8s-bot still needs to wait its turn in submit queue and pass tests.
 
 ![Bot Notification for Approval Mechanism](images/bot_notification_for_approval_selection_mechanism.png)
 
-**Final Notes**
+## Configuration options
+
+See the [Approve](https://godoc.org/k8s.io/test-infra/prow/plugins#Approve) go struct for documentation of the options for this plugin.
+
+See also the [Lgtm](https://godoc.org/k8s.io/test-infra/prow/plugins#Lgtm) go struct for documentation of the [LGTM](#lgtm-label) plugin's options.
+
+## Final Notes
 
-Obtaining approvals from selected approvers is the last step towards merging a PR. The approvers approve a PR by typing `/approve` in a comment, or retract it by typing `/approve` cancel. 
+Obtaining approvals from selected approvers is the last step towards merging a PR. The approvers approve a PR by typing `/approve` in a comment, or retract it by typing `/approve cancel`.
 
 Algorithm for getting the status is as follow:
 
-(1) run through all comments after newest commit to obtain latest intention of approvers
+1. run through all comments after newest commit to obtain latest intention of approvers
```

Comment:
I think this is wrong, and that the plugin looks at all comments even those before the latest commit (based on observation of behavior and looking at the code), but I'm not certain and I'm not making things worse here.

Reply:
You are correct, the plugin looks at all comments and considers the latest related command from each user (so that I can `/approve`, then `/approve cancel`, then `/approve` again and get the expected behavior at each step of the way).